lib/joke.rb

```diff
@@ -0,0 +1,12 @@
+class Joke
+attr_reader :id
+attr_accessor :setup, :punchline 
```

Comment:
No reason for these to be attr_accessors. You're not being asked to set these values from outside of the class.